### PR TITLE
Add evaluation controller

### PR DESCRIPTION
**Pull request by BrainMaestro**
Completes evaluation in #62.

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Approved

File: app/Listeners/EvaluationSubmitted.php [hunk 1]
```diff
@@ -24,13 +27,17 @@ public function __construct(EvaluationRepository $repository)
...
-        foreach (func_get_args() as $evaluation) {
+        foreach ($evaluations as $evaluation) {
             $this->repository->incrementEvaluation(
                 $evaluation['id'],
                 $evaluation['column']
             );
         }
+
+        if (!is_null($comment)) {
+            $this->commentRepository->createComment($semesterId, $questionSetId, $comment);
```
Comment (BrainMaestro) [inline app/Listeners/EvaluationSubmitted.php]: Fixes #97

**Comment by tenorz007** [comment]
Why using evens and listeners? @BrainMaestro

**Comment by BrainMaestro** [comment]
Hard to explain, but it's a cleaner way to handle it. Other actions could listen in on the event in the future.

**Comment by tenorz007** [comment]
Other actions like what, we are still doing the same many DB calls as if
you just called the Repository directly. SO?

**Comment by BrainMaestro** [comment]
Like pushing the change to the UI. What many DB calls? Weren't you part of #68? It's currently not possible to do it in one DB call. I tried, so did @SkaeX

**Comment by tenorz007** [comment]
Are you listening to what I said. In case you didn't, I said "we are still
doing the multiple DB calls using the events" which would be same as
calling the repositories, so why not just use it.

**Comment by BrainMaestro** [comment]
We're separating concerns. It's cleaner that way. You can change it if you want

**Comment by tenorz007** [comment]
it's not about "You can change it if you want." I just though there was some grand scheme to it or some way it helps solve our issue.

**Comment by BrainMaestro** [comment]
It helps solve the queuing issue and it's clean. Because that is an event. It just makes sense. Don't know how else to explain.

**Comment by tenorz007** [comment]
Approved

**Pull request merged**